init/cli/logging.go

```diff
@@ -72,13 +82,28 @@ func (f *FileLoggers) SubProcessLogger(name string) launchlib.CreateLogger {
 }
 
 func (f *FileLoggers) OpenFile(path string) (*os.File, error) {
+	if _, ok := f.rotated[path]; !ok {
+		if _, ok := f.flags.(*truncatingFirst); ok {
+			rotate(path)
+			f.rotated[path] = true
+		}
+	}
 	file, err := os.OpenFile(path, f.flags.Get(path), f.mode)
 	if err != nil {
 		return file, errors.Wrapf(err, "could not open logging file '%s'", path)
 	}
 	return file, nil
 }
 
+func rotate(path string) {
+	limit := 5
+	_ = os.Remove(path + "." + strconv.Itoa(limit))
+	for i := limit; i > 0; i-- {
+		_ = os.Rename(path+"."+strconv.Itoa(i-1), path+"."+strconv.Itoa(i))
+	}
+	_ = os.Rename(path, path+".0")
```

Comment:
Spacing looks odd here, isn't there an auto-formatter that's supposed to automatically check and fix these things?

Reply:
This was the autoformatter. I think when a function has multiple arguments, then the spaces are removed. You can see on line 100 above, the function takes one argument and there are spaces.